python/pylibcudf/pylibcudf/tests/conftest.py

```diff
@@ -17,13 +17,23 @@
 from utils import ALL_PA_TYPES, DEFAULT_PA_TYPES, NUMERIC_PA_TYPES
 
 
-# This fixture defines the standard set of types that all tests should default to
+def _type_to_str(typ):
+    if isinstance(typ, pa.ListType):
+        return f"list-{_type_to_str(typ.value_type)}"
+    elif isinstance(typ, pa.StructType):
+        return f"struct-{'-'.join([_type_to_str(typ.field(i).type) for i in range(typ.num_fields)])}"
```

Comment:
I like `struct[int, float]` more than `struct-int-float` because it makes the hierarchy clear.
```suggestion
        return f"struct[{', '.join([_type_to_str(typ.field(i).type) for i in range(typ.num_fields)])}]"
```